config/management/src/validator_config.rs

```diff
@@ -53,6 +70,11 @@ impl ValidatorConfig {
             .append_prod_protos(validator_network_key, HANDSHAKE_VERSION);
         let raw_validator_address = RawNetworkAddress::try_from(&validator_address)
             .map_err(|e| Error::UnexpectedError(format!("(raw_validator_address) {}", e)))?;
+        let enc_validator_address = raw_validator_address
+            .encrypt(&root_key, key_version, &sender, sequence_number, addr_idx)
+            .map_err(|e| Error::UnexpectedError(format!("(enc_validator_address) {}", e)))?;
+        let raw_enc_validator_address = RawEncNetworkAddress::try_from(&enc_validator_address)
+            .map_err(|e| Error::UnexpectedError(format!("(raw_enc_validator_address) {}", e)))?;
```

Comment:
I know this seems to be the Rust way to do it, but could we attach the error handling of a poorly formed NetworkAddress conversion to the converting function.  e.g. use RawEncNetworkAddress::from(&enc_validator_address), and put the mapping of error there to cut down on places we may reuse it.